src/qt/transactiontablemodel.cpp

```diff
@@ -722,7 +722,7 @@ static void NotifyTransactionChanged(TransactionTableModel *ttm, const uint256 &
     notification.invoke(ttm);
 }
 
-static void ShowProgress(TransactionTableModel *ttm, const std::string &title, int nProgress)
+static void ShowProgress(TransactionTableModel *ttm, MAYBE_UNUSED const std::string &title, int nProgress)
```

Comment:
`title` appears to be unused as well. I think a better approach is to remove the parameter name.